Tests/GitHubAPI/operationIdsPath

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Should this file maybe go to `operationIdsPath.json` instead of not having an extension?

Reply:
Agreed! Updated, thanks!